routes/movies.js

```diff
@@ -0,0 +1,111 @@
+const Celebrity = require('../models/celebrities');
+const Movie = require('../models/Movie');
+const router = require('express').Router();
+
+router.get('/movies/new', (req, res, next) => {
+  Celebrity.find().then(celebritiesFromDb => {
+    res.render('movies/new', {
+      celebrities: celebritiesFromDb
+    })
+  }).catch(err => {
+    next(err)
+  })
+})
+
+router.get('/movies', (req, res, next) => {
+  Movie.find().populate('cast').then(moviesFromDb => {
+    res.render('movies/index', {
+      movies: moviesFromDb
+    })
+  })
+})
+
+router.post('/movies', (req, res, next) => {
+  const {
+    title,
+    genre,
+    plot,
+    cast
+  } = req.body
+  Movie.create({
+    title: title,
+    genre: genre,
+    plot: plot,
+    cast: cast
+  }).then(newMovie => {
+    res.redirect('/movies')
+  })
+})
+
+router.post('/movies/:id/edit', (req, res, next) => {
+  const movieId = req.params.id;
+  const {
+    title,
+    genre,
+    plot,
+    cast
+  } = req.body
+  Movie.findByIdAndUpdate(req.params.id, {
+    title: title,
+    genre: genre,
+    plot: plot,
+    cast: cast
+  }).then(c => {
+    console.log(c.cast)
+    res.redirect('/movies')
+  }).catch(err => {
+    next(err)
+  })
+})
+router.get('/movies/:id/edit', (req, res, next) => {
+  const movieId = req.params.id
+  Movie.findById(movieId).populate('cast')
+    .then(movie => {
+      console.log(movie)
+      Celebrity.find()
+        .then(celebrities => {
+          let options = '';
+          let selected = '';
+
+          celebrities.forEach(actor => {
+            selected = movie.cast.map(el => el._id).includes(actor._id) ? ' selected' : '';
+            options += `<option value="${actor._id}" ${selected}>${actor.name}</option>`;
+          })
+          res.render('movies/edit', {
+            movie,
+            celebrities
+          })
```

Comment:
I think you could remove this if you want to add selected to the actors in the backend (like you did in line 72). If you want to use an hbs helper function, then you keep these lines and remove lines 78-81